src/v2/Apps/Order/Components/SetPaymentByStripeIntent.tsx

```diff
@@ -26,6 +27,7 @@ export const SetPaymentByStripeIntent: FC<Props> = props => {
               input: {
                 id: props.order.internalID,
                 setupIntentId: setupIntentId,
+                oneTimeUse: saveAccount === "true" ? false : true,
```

Comment:
#minor: Guess this is what prettier formatted. I was thinking it may be more concise to just say
```typescript
oneTimeUse: saveAccount !== "true"
```